stdpopsim/catalog/AraTha/dfes.py

```diff
@@ -10,8 +10,8 @@
 
 
 def _HuberDFE():
-    id = "Gamma_H18"
-    description = "Deleterious Gamma DFE"
+    id = "GammaAdditive_H18"
+    description = "Deleterious additive Gamma DFE"
     long_description = """
```

Comment:
Ah - could you update the long description, also? How about
```
    Additive, deleterious DFE from Huber et al. (2018) for Arabidopsis, 
    estimated using <software> from A. lyrata as a Gamma distribution
    of deleterious effects. Parameters are from Supplementary Table 4,
    the "genome-wide, additive-only model for A. lyrata". The DFE for
    lyrata (rather than thaliana) is provided due to challenges with
    simulating selfing.
```

Reply:
ah, I see `<software>` wasn't some named software package, they just did it using Methods from the paper. I rephrased this to "using the SFS"